build/images/Makefile

```diff
@@ -22,12 +22,11 @@ docker/index.txt: ../../docker/index.yaml
 
 .PHONY: chartmap
 
-chartmap:
-	@echo "manifest,chart,image" >> chartmap.csv
-	@tac chartmap.csv > chartmap-tac.csv
-	@mv chartmap-tac.csv chartmap.csv
+chartmap: index.txt
+	@echo "manifest,chart,image" > chartmap.csv
+	@find . -type f -name chartmap.csv -exec cat {} \; | sort -u >> chartmap.csv
```

Comment:
Not sure if I understand the logic here. Old logic was:
* write header line to the end of `chartmap.csv`
* reverse lines in this file so that header goes first

New logic is:

* create new `chartmap.csv` consisting only of header line
* find all files named `chartmap.csv` in current directory, recursive and write sorted content into `chartmap.csv`

This raises many questions ... Are there any other `chartmap.csv` files in current dir? Won't bash lock `chartmap.csv` for wiriting, before `find` will call `cat` to read from this file? My understanding that even if bash does not complain on reading from locked resource, `chartmap.csv` will still consist of only header...

Reply:
@mtupitsyn there are multiple chartmap.csv files created in sub-directories, in ```extract.sh```.

Reply:
Ah, you're right -- good catch. I also see the headers duplicated in the top-level chartmap.csv to your concern. I'll add a find qualifier to address, test, and put up another commit.

Reply:
New commit up to address @mtupitsyn.